src/main/java/seedu/address/model/house/NonLanded.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.house;
+
+import seedu.address.model.house.House;
+import seedu.address.model.person.Person;
+
+public class NonLanded extends House{
+    public final Block block;
+    public final Level level;
+    public final UnitNumber unitNumber;
+
+
+    /**
+     * Constructs a {@code Level}.
+     *
+     * @param block      The block of the house.
+     * @param level      The level of the house.
+     * @param postalCode The postal code of the house.
+     * @param street     The street of the house.
+     * @param unitNumber The unit number of the house.
+     */
+    public NonLanded(Block block, Level level, PostalCode postalCode, Street street, UnitNumber unitNumber) {
+        super(unitNumber, postalCode, street);
```

Comment:
Can change to something like this `super(postalCode, street);`
The rest of the class variables looks good 👍